### DISCUSSION TITLE
Toughts on Mythic system

### DISCUSSION THREAD

**Discussion by kasvati** [chromiecraft/chromiecraft#7337] [Suggestions]
### Feedback on the Mythic System

Hey there,

I have never really shared my opinions much on the matter because I didn't think this feature would see revival in any major capacity.

For the record, I'm a raider, both in guild raids and PUGs, from Kara to Hyjal and everything in between, multiple times a week. I mention this because my perspective is partly shaped by this experience.

First, I'm going to roughly describe the Mythic system with its pros and cons, and then explain each one of them in their own section.

---

### Current Behavior of the Mythic System:

The Mythic system introduces a new difficulty mode with more challenging optional content and a reward structure behind it. Killing each boss on mythic difficulty rewards points, which you can use to exchange for items related to that content. Currently, there are only two raids using this system: Gruul's Lair (2 encounters) and Magtheridon's Lair (1 encounter).

#### **Pros:**
- Acts as a pity system or protection from "bad luck"

#### **Cons:**
- System was not made with Black Temple in mind
- Inauthentic to the original
- Deterministic way to obtain items
- Difficulty
- Player retention

I might seem overly negative, but let's explore each of these points.

---

#### **Acts as a pity system or protection from "bad luck"**

Fair RNG is, in fact, not fun for the average player. This is due to both a stronger bias towards negative emotions and the human inability to internalize statistics. While I believe (perhaps naively) that most people can understand the basics of statistics, I don't think they would describe the current RNG as "fun." This is what the system partially solves or rather tries to compensate for—the lack of biased RNG towards players. (FYI: I will most likely elaborate on this later.)

---

#### **System was not made with Black Temple in mind**

The system itself was not made for a 9-encounter instance, which makes this very unbalanced. In fact, assuming 25 players kill all 9 bosses, it would inflate the item count by 22.5 deterministic items each lockout. For comparison, current T4 has hypothetically 7.5 items of inflation, but a more accurate number would be 5, as Magtheridon tends to be skipped.

UPDATE: It has been clarified that the amount of items gained should be similar to T4 raids so whenever i reference this just work with values between 5 and 7.5 which is still a lot btw.
https://github.com/chromiecraft/chromiecraft/discussions/7329#discussioncomment-10530866

#### **Inauthentic to the original**

This is a fun feature that was not in the original and is not authentic in either implementation or spirit. Currently, the server is advertised as a blizzlike (with some QoL) progression server, but what we get is a funserver with extra steps. With the amount of inflated experience, gold, and most importantly, items, the server shouldn't be considered as such anymore. While I cannot speak for all players, old and new, there is a sentiment that we signed up for a blizzlike progression server, and it’s being transformed into a funserver with handouts after considerable time already spent.

#### **Deterministic way to obtain items**

This is a very powerful feature of the system. It enables a player to choose a guaranteed item, no matter the rarity or power of the item—the cost is the same. Most people don't realize how powerful this feature is. In fact, I would say it's too powerful. This feature is exactly why the Mythic option should never be an option. It would make the mode mandatory. After all, 22.5 handpicked extra items each lockout is too big of a reward; in fact, even the 5 from Mythic Gruul's Lair is way too much of a reward. Virtually no one is running Normal Gruul unless they are forced to. On top of that, I doubt many players would feel happy picking a less optimal way to gear up or to hamstring their own guild's progression.

#### **Difficulty**

In every sense of the word "difficulty."  
With higher difficulty comes a higher risk of failure and punishment (wasted time, consumes, etc.). So naturally, players will adapt and either get good or get removed. There is seemingly no issue at first; however, difficulty can also be translated as a "barrier to entry." Increased difficulty will always promote gatekeeping and segregate the player base. An already small pool of players will thin even more as raid leads will want to get the highest quality players and filter out bad players—or even worse, new players. If new players are gatekept away, they will never get an opportunity to learn (and they need to learn a lot, as increased difficulty will also increase the amount of things they need to learn) or get into current content raiding. On top of that, difficulty will be mandatory because the reward is simply too great.

Next is difficulty in terms of implementation. While the store and points are implemented, what I'm really concerned about is the raid itself. This feature would increase the workload for the devs considerably, and given their mishaps in recent brackets, I would rather they focus on one game difficulty and get it right than end up with two unfinished ones.

#### **Player retention**

Item inflation will speed up gear progression, and in this case, insanely so. This will decrease the lifespan of the bracket, and players will be "done with it" way sooner. Veterans might quit sooner or go on a break (or quit during a break), again decreasing the player pool even further, and this time skilled players. In turn, this will also create pressure to release the next bracket sooner, giving the devs less time for subsequent brackets, ZA, and SWP.

---

### Conclusion

The Mythic system is a difficulty that has a massive impact on the game, especially if implemented in the latest raid tier. The combination of reward and difficulty will create a toxic feedback loop of getting rewarded for an inferior raiding experience that will demoralize the player base even more. It will create more work for devs and more possible points of failure for the bracket, with potentially huge costs in the future. But hey, on the flip side, everyone will have their own Skull of Gul'dan.

### Suggestion

Just scrap it and focus on making the raid fun, as that is what matters anyway.

But on a serious note, if there is a consensus that the current RNG is an issue, then a compromise would be a system that actually addresses the issue at its core—and that’s RNG biases in favor of players. For example, keep track of loot for each character for like 6 months, and based on that data, you can aggregate it across the raid group and influence drop rates on each successful encounter. Obviously, this would also have some unwanted effects or trade-offs, but I think it’s way better than the "mythic mode" for solving the RNG issue. And most importantly, keep it secret, or at least do not advertise or mention the feature. Let the players believe and be happy for their own incredible "luck." Btw, this method of RNG rigging is pretty common across many games.

---

### My Own Thoughts on the Matter

While I focused on the negatives a lot and painted them in very bleak colors, it's not without merit I have been on the internet way too much and played WoW also way too much so I can tell a fuck up when i see one. Maybe those effects will not manifest to the extent I envisioned them, but they are still there. The feature itself shows quite clearly its scope of impact and its direction the only unknown is its magnitude. I could have also mentioned issues with how players perceive the value of items, especially in WoW, or gone in-depth on the difficulty of tuning raid encounters, but it was getting too long, and I wanted to focus on points that are more important and have the ability to compound on each other. 

When the announcement dropped, I didn't even know about it until someone from my guild mentioned it. And when I read it, I felt disappointed—not from the feature itself, but because it was announced so nonchalantly as if it were just some bonus thing. But any person who knows a thing or two about WoW would immediately see the implications and impact. So, either the devs were clueless about the problems that the current system creates, or they simply don't give a fuck.

opinions, thoughts, insults leave below tyvm

**Comment by Nyeriah** [comment]
You seem to labour under the misapprehension that the Mythic system is there for the sole purpose of giving players an additional piece of gear, when in fact what it tries to address is the issue that it is hard to strike balance when it comes to difficulty to reach the point content is accessible and enjoyable to (if not everyone) as many people as possible - and that has been a recurring complaint among players and clearly voiced in our past review.

You seem to understand the importance of not gatekeeping players behind difficulty given your response, so I believe you will also be able to understand that every effort made towards allowing more players to actually play (and enjoy) the game is welcome.

It's much preferrable to us that players actually get to enjoy the game and do content even if it a lower difficulty than quit out of frustration and burn out. So at this point, getting a single piece after running the same bosses every week for the Nth time is not very concerning and certainly won't impact the progression as much you believe.

The gear reward, after having completed countless raids, is merely a bonus in this scenario. And IMHO it's very fair considering the effort/time investment raiding at top difficulty is going to be. Also, I have no idea where some of your numbers come from, e.g "22.5 handpicked extra items each lockout is too big of a reward", which I can say with certainty... has never and will never be the case.

**Reply by kasvati** [reply]
1. No im not under misapprehension, in fact because of that I'm aware of the issue. Its not balancing in isolation nor reward structure in isolation but in fact in combination as a whole. And I did mention that explicitly. And I know balancing is difficult but frankly speaking, Mythic option with an extra reward structure as an advantage is not an option. its the base line in fact the normal is the option in this relationship.
So what you are making is that content that's meant to be for more hardcore audience is in fact expected baseline for everyone.
If there is such a huge demand for more difficult content then just make Mythic and strip out the reward structure and see how many people participate. Because only then it would be truly optional.

2. "In fact, assuming 25 players kill all 9 bosses, it would inflate the item count by 22.5 deterministic items each lockout." I think the numbers is clear. it was made with the announcement which said that the costs will be the same. so if boss gives 1 point and item costs 10 then one raid group of 25 players would generate points in worth of 22.5 items. Now this is currently wrong as you clarified yourself said that this should be in fact inline with T4 reward structure which would be using the same logic, which roughly speaking would be 5 for Gruul's lair and 2.5 for Maggy. I have added a little note just below it when my guildie alerted me to. The reason for that is you people can simplify statistic for them now its easier to visualize the numbers when 50 raiders that complete both maggy and gruul on mythic would generate 15 deterministic items in a lockout. Now the bad thing is that items are not equal to each other and 1 deterministic item is i would argue multiple times valuable than 1 random item. Frankly speaking to me it seems like a lot.
Now I could have delved into whole what is the value of items conversation but that would take too much time for now. However you can count on that this type of system devalues rare and powerful items because they will become most common by the fourth of fifth week.

3. Of course if the scenario is that you run through the raid so many times that you don't need the gear then the reward is nearly irrelevant. But that's not the only scenario in fact i would say its quite uncommon especially for current tier raid. Such as upcoming Black Temple. And if the people thought that the reward is only bonus why people still talk about it. Whenever i was debating this system with anyone. The point system and items were most of the time if not always the most vocal point of discussion.

Honestly I'm sorry if I feel combative or pushy but sometimes i feel like i need to explain things that should be obvious so i feel stupid for even explaining it.
Maybe it was fault on my end in the communication of the original post.

**Reply by heyitsbench** [reply]
> then just make Mythic and strip out the reward structure and see how many people participate.

You must have missed the who knows how many people saying SSC through Hyjal *were* just mythic without the rewards and how many people very much disliked that structure.

**Reply by kasvati** [reply]
1) effort exerted didn't match the reward
2) they went from deterministic reward that is very powerful and feels very good to one that feels comparatively worse
3) Bugs, and hyjal as a raid sucks design wise on top

so its natural that people were pissed, it would be weird if they weren't
I was just making the suggestion make it more apparent how much reward is important in this conversation

**Reply by Nyeriah** [reply]
You must have missed the part it says **it will take the same amount of time to acquire items as it did before**. Information has been rectified in the announcements posted. 

Despite the amount of bosses, the cost will be scaled up. So no, there won’t be 22 new items every lockout and nobody is going to get a free item every raid… that’s just absurd…

**Reply by kasvati** [reply]
> UPDATE: It has been clarified that the amount of items gained should be similar to T4 raids so whenever i reference this just work with values between 5 and 7.5 which is still a lot btw.
> [#7329 (comment)](https://github.com/chromiecraft/chromiecraft/discussions/7329#discussioncomment-10530866)

> Now this is currently wrong as you clarified yourself said that this should be in fact inline with T4 reward structure which would be using the same logic, which roughly speaking would be 5 for Gruul's lair and 2.5 for Maggy.

No I'm aware its absurd, and I have mentioned the clarification twice now. I was trying to explain to you why I wrote those numbers the way i did.
However inflation of 7.5 points of items and lets be real its going to be the most powerful and rarest items per raid group and per lockout is in fact also a lot.(that's with scaling in mind assuming its going to be kept in line with T4)

**Comment by Nyeriah** [comment]
Also some off-topic but: 

> the server is advertised as a blizzlike

It is not, you won't find that term being used anywhere in any of our medias (website, discord, etc.)

**Reply by kasvati** [reply]
This is my opinion the whole thing, not a statement of a fact but my perception, my concerns and my rationale behind those concerns.
Regardless of whether chromie explicitly used the term or not, its not a stretch to make assumption that this a blizzlike server with progression that idea would come to me implicitly since there is no mention of custom items or raids as far as i remember maybe in fine print i guess i should get a lawyer next time before i play on a private server.

Obviously I'm joking here but I'm trying to voice my opinion in good faith for you guys and what I get for it is a wordplay. Come-on were not 5yo.

**Reply by Shardaz** [reply]
It was advertised like this when a lot of people joined. And changing this after those people invested multiple hundreds or thousands of hours, is, sorry for the term, a dick move. I love the server and the philosophy it has (had). I, and I believe many else, wanted to live through the whole thing on the same conditions I signed up for long ago. I wanted to play Wotlk with a progression system. Other people that demand changes seemingly want to play a different game. If there are two sides, why cater to the people that actually dont like the game instead of the people that do?

**Reply by Nyeriah** [reply]
It hasn’t been announced as blizzlike dating from the earlier stages of vanilla. As soon as we came to realize it could be misleading we’ve removed such term.

**Reply by andovcr** [reply]
> It hasn’t been announced as blizzlike dating from the earlier stages of vanilla. As soon as we came to realize it could be misleading we’ve removed such term.

Please do not say such things without actually checking your source material:
<img width="748" alt="image" src="https://github.com/user-attachments/assets/224add5b-5949-4779-a506-0800401d231a">

[https://www.chromiecraft.com/en/rates-n-changes/](https://www.chromiecraft.com/en/rates-n-changes/)

<img width="245" alt="image" src="https://github.com/user-attachments/assets/dbd280fd-a65d-4a33-850b-3e14604490e0">

[https://www.reddit.com/r/ChromieCraft/](https://www.reddit.com/r/ChromieCraft/)

**Reply by Nyeriah** [reply]
We have stated reiterated times we are not a blizzlike server and we don't aim to be so (from the fact we are a progressive realm locked at level ranges alone, which is incompatible with "blizzlike" by definition).

It's unfortunate that the Reddit community still holds that description. I don't use Reddit, but I will be contacting the one responsible for it so it can be updated.

"Blizzard-flavoured" isn't blizzlike, we didn't intend it to be a synonymous term when it was used. I will see if I can get it removed as well given it may be misleading to some.

**Reply by Belodina** [reply]
Ah, thanks for pointing that out Andov, they should probably update the reddit page even though it is mostly devoid of activity. I believe the first screenshot is entirely true and relevant though. Besides, the addition of a mythic mode is not the issue, only the rewards (of which no custom items are being considered which seemed to be the issue previously). So whether or not it is Blizzlike seems to be a moot point.

Mythic as a concept is blizzard-like, even if it is not true to the original 3.3.5 experience though.

**Reply by Nyeriah** [reply]
@andovcr Thanks, both references were removed so hopefully there's no further confusion about this.

**Comment by Never6** [comment]
I like mythics because, imo there should be no scenario where I do a raid for 8months straight and count on one hand how many particular drops I have seen in that time. 
"its not blizzlike." I agree but this server hasn't been blizzlike for a LONG time. You are playing level 70 with WoTLK talents. its not blizzlike and is never going to be
The gear is going to be all irrelevant once we hit 80 anyway,

**Reply by SaadChaudhry002** [reply]
Brother why breathe when you will be irrelevant once you die anyways
actually worst take ive read in months

**Reply by Belodina** [reply]
Comparing a video game to real life is a worse take, there is no need to be toxic here, please.

Additionally, with the mythic system implemented, there is more development for a module for AzerothCore that will likely be very popular with people running their own servers.

**Comment by Shardaz** [comment]
Considering the impact of the mythic mode, I would strongly suggest a real survey pointing towards mythic raids.

- How do you feel about the return of mythic raids in the **current** Tier?
- How do you feel about mythic raids in the **past** Tiers?
- Do you like the current point system?
- If yes, how many weeks of playing should it take to get an item?
- If no, what would be a better reward system?
- Should there be any reward system at all?
- etc etc...

These are all questions you need to ask your playerbase. And imo, if there is a fair amount of people against (parts of) the system, it should NOT be implemented. Please cater to people that actually like the game, not to people that don't.

On a side note: Why was the system discontinued after Gruul/Mag? Ah yes, people hated it.

**Reply by Never6** [reply]
> 
> On a side note: Why was the system discontinued after Gruul/Mag? Ah yes, people hated it.

no, people hated it because originally they were going to get custom gear for doing mythic dungeons/raids

**Comment by vvranceanu** [comment]
Balancing the game and implementing features around paepaegas that cant press 4 buttons in correct order and are not motivated to learn that is simply bonkers.

You decided to cater around 2 week butterflies and fucked people that joined 2+ years hoping for a wotlk-style progression server.

Instead of focusing on quality of releases that is the root cause of all complains, you went with implementing bread and circus.

You have no principles and no dignity.

**Comment by veredis1** [comment]
I am surprised that there was no poll about such a big change, would liked to see one there. Bench said above that many people told there concerns how they disliked the system but people that are happy with that are not such a loud voice in Discord etc. I liked the system and now I ll probably feel robbed for a real Illidan kill. And my biggest fear is that it ll be so waterdown to a point there I dont really feel like gaming anymore. Also I am not sure if guilds will bother with mystic, in t4 content only some did it. It was (like it seemed to me) gaining popularity during t5 after nerfs and better gear.

Also the mythic points will bring more items into the raid even from bosses the raid couldnt even do. And like said above the current system (gruul/mag) for bt will give u tons of points. The extra Items will make the raids also even easier in the progress.

**Reply by AcetylGer** [reply]
Mythic will not bring any items "from bosses the raid couldnt even do". To unlock the items from the vendor you need to do a full 9/9 clear (or buy a carry for an illidan kill).

So no, nobody is robbing you of a real Illidan kill, you even get two of them progressing normal till Illidan and then taking on the challenge of clearing the whole raid on mythic. And then after getting your mythic Illidan kill you get rewarded with being able to spend your gear.

**Comment by Belodina** [comment]
There seems to be some echo chamber regarding opposition to the Mythic System, in which the absolute worst-case scenario is being considered as the only one. 

Patience and trust are in order. The devs and project leads have shown themselves capable of running the server, and have access to more information than any one individual or guild has. If they believe that the addition of the mythic system is healthy for the server, then there is a good chance they are right. A lot of the fear surrounding the return of the system appears to be largely based on misinformation, no information or speculation as to how the system _may_ work when in reality we do not know yet.

Personally, the addition of the mythic system and bad luck protection are incredible ideas. Having run Hyjal weekly for three months now, I am still missing items, which is a result of Blizzard's original game design and doesn't feel great. The addition of bad luck protection in the form of mythic points is wonderful, as I have performed consistently and at a high level for weeks and shouldn't have to be shafted by RNG. The server hasn't been advertised as Blizzlike for a long time and the addition of an optional mode for those that enjoy the challenge without rewards is nonsensical.

The purpose of the server is to script the content and fix bugs. If the dev team feels the addition of mythic is doable while maintaining their current ability to fix the content, then you simply have to trust them, or contribute and help code.

tl;dr Mythic is not as world-ending as you paint it to be, there are major upsides which you are not considering and the system is likely to be a great addition to the content.

**Reply by kasvati** [reply]
> Maybe those effects will not manifest to the extent I envisioned them, but they are still there. The feature itself shows quite clearly its scope of impact and its direction the only unknown is its magnitude.

I have never said that this is the end of the world, but from my PoV chromie is investing resources into a feature that is more harmful than good in its fundamental philosophy. If the post informed about about my concerns then it has fulfilled its purpose.

I have patience but i do not trust devs to do a good job. Not after hyjal. And this announcement only strengthened that belief.

Mythic system as it stands is bad for reasons i have mentioned in the post. If you want to implement the system to remedy to bad RNG then its also bad because it also brings all the negative effects along with it. To solve the issue with RNG, solution should address that directly and not in round about way of compensation. Now are there ways to improve on the mythic system to make less destructive ? certainly there are. But you cant identify those issues without talking about them and making them apparent.

Why would I invest my time into a project that in my eyes tries to sabotage itself. In fact this feedback is also a form contribution but trust me at this point I'm so demoralized I don't even feel like doing anything more than that.

Also please don't disregard opinions of others that you don't agree with as an "echo chamber". It's very disrespectful.

**Reply by Belodina** [reply]
There was no disrespect directed towards your or anyone's opinions, and I'm sorry you feel that there was. It is my perception that the backlash to the announcement for Mythics returning has been echo chamber-y and hasty, as a lot of assumptions have been made about how it will be implemented and the negative consequences thereof with minimal information or facts being considered in any capacity. The announcement has been out for only two days so far.

I cannot help that you do not trust the devs to implement the system, but that sadly does not assist in giving any weight behind your opinions in regards to actioning any form of meaningful change. Admitting something like that makes it less likely that the opinions you bring forward will be accepted in good faith if they are made without.

> If the post informed about about my concerns then it has fulfilled its purpose.

I worry that the consequences of voicing misinformed opinions and/or views that have not been given thorough consideration are harmful to the project in a manner that seeks to do the opposite. Instead of trusting the people who have an overhead view of the negative and positives of the addition of such a system (and likely have put far more considerate thought into its reintroduction) we are instead arguing in circles and causing unnecessary division in the community, all while being combative with people who are working to provide an enjoyable experience.

> So, either the devs were clueless about the problems that the current system creates, or they simply don't give a fuck.

I believe we are all here because we agree with the values and end goal of the project of ChromieCraft, and are not here to dictate how such a project achieves its end goal, but instead, stay positive and open-minded regarding the direction the project takes. While I agree that the staff have made mistakes in the past, which are useful lessons, I do not agree with the negative manner in which this is being approached.

All I am asking is that you hold hope and patience close at hand and consider the angle from which the team is viewing this whole situation. And that you assist in stifling the flames of division which might arise in your dungeon groups, guild chat, discord or on GitHub instead of fanning them.

I'm not asking you to change your mind (while I disagree with your points personally) as this is a valid suggestion and _should_ be considered with equal weight to anyone else's opinion, to disrespect the team in such a manner is not conducive to the project in any manner and should not be encouraged.

**Comment by krutoj87** [comment]
I like the mythic system and I hope it stays for WotLK aswell. There has already been written a lot about this topic by other commenters, but one thing wasn't touched on yet:

> Player retention
Item inflation will speed up gear progression, and in this case, insanely so. This will decrease the lifespan of the bracket, and players will be "done with it" way sooner. Veterans might quit sooner or go on a break (or quit during a break), again decreasing the player pool even further, and this time skilled players. In turn, this will also create pressure to release the next bracket sooner, giving the devs less time for subsequent brackets, ZA, and SWP.

I don't think that's true. Players don't necessarily quit, when they have all the gear. They quit when they stop having fun. Having collected BiS gear already can lead to this sure. But I think on a server where most people seem to have armies of alts, I don't think that's a big of a problem. On the other hand if people raid the same content for months, for the last elusive items and they never drop, this can also ruin the fun. 
It's impossible to say if mythic points will affect player retention in a positive or negative way. Both is possible. It's even possible, that it won't affect player retention at all one way or the other. Thus this point is pure speculation and not a real argument.

**Reply by kasvati** [reply]
Finishing the content early will always lead to some level of hiatus on the players side, just hoping that everyone will make an alt is strong assumption. People tend to leave or take a break during content droughts and there is nothing wrong with that, if we had larger population of the server the issue wouldn't be that big. Also you can just check the history of content drought on retail if you want confirmation on this behavior.

RNG sucks and i think i have addressed it in pretty elegant way. Altho i would rather to keep it basic as possible.
In suggestion I made it apparent that it should be solution to the issue and not some sweeping system that has significant impact on the game.

**Reply by krutoj87** [reply]
> Also you can just check the history of content drought on retail if you want confirmation on this behavior.

You say it yourself. Retail also has falling player numbers a few weeks into the content without a bad luck protection system. Just like the CC playerbase dropped extremely during the SSC, TK and Hyjal phase. There hasn't been any bad luck protection here either. 

Also you missed my point. I'm not saying everyone will be playing an alt when they are geared. What I'm saying is there are so many variables that might cause players to leave or not, that it's impossible to say if the mythic system will have a negative or positive effects. You are just speculating that the effect will be negative, you have no data to prove this.

**Reply by kasvati** [reply]
People quit not because lack of "bad luck protection" but because they are not having fun.

So the question is why are people not having fun ? and then one of the possible reasons could be lack of "bad luck protection" but it also could have nothing to do with the reason they quit. In this case it is a speculation because you don't have anything that would confirm or even give logical meaning that cause of the data is lack of "bad luck protection". It could be a whole myriad of things. Some people are casual gamers that buy the expansion to play for two weeks and quit until next one(and in my opinion i would guess that's a huge chunk of people that quit early). 

What I'm saying that system negatively impacts the retention because people will get their BiS more quickly as the nature of the system is to create more items in the economy. So naturally that would be the case because. One of the aspects of the game is gearing a character. For people who enjoy that this aspect of the game(like me) would end quicker with this system in place. Thus removing one of the things that motivate people to play. So yes this will always have negative effect on player retention. HOWEVER it wouldn't be that big of an issue if we had more healthy population on chromie. But alas we dont.

also yes some people quit out of frustration i get that, but people running out of fun things to do quit for sure.

**Reply by vvranceanu** [reply]
`On the other hand if people raid the same content for months, for the last elusive items and they never drop, this can also ruin the fun.` if you're that lost that you obsess about 2 items that give you 2% increased performance each that you think its justified to modify the very structure of the game YOU signed up to play, then go touch grass.

`It's impossible to say if mythic points will affect player retention in a positive or negative way. Both is possible. It's even possible, that it won't affect player retention at all one way or the other. Thus this point is pure speculation and not a real argument.`
How about, oh, i dont know, not change it then and let people that joined wotlk to get what they expect of a wotlk server? No one joins vanilla servers and start crying there are no emblems to buy gear with.

**Reply by krutoj87** [reply]
>  if you're that lost that you obsess about 2 items that give you 2% increased performance each that you think its justified to modify the very structure of the game YOU signed up to play, then go touch grass.

We aren't talking about me here we are talking about the general player base and this player type exists in the player base. Maybe try to make arguments instead of trying to discredit me as a person.

> How about, oh, i dont know, not change it then and let people that joined wotlk to get what they expect of a wotlk server? No one joins vanilla servers and start crying there are no emblems to buy gear with.

This is a nonargument, because different people expect different things and many people specifically asked for the mythic system to come back. No matter what you do some people will be unhappy.

**Reply by krutoj87** [reply]
> also yes some people quit out of frustration i get that, but people running out of fun things to do quit for sure.

1. Running out of fun things to do isn't the same as having got all the equipment from one raid. You can be fully geared and still have fun with the game. You are just pushing for the narrative, that the game is over once you are geared, because it suits your argument. It's true for some and it's wrong for others. 
2. What makes more people quit, being frustrated with the game or having obtained all the equipment from one raid? And if you say you can tell one or the other makes more people quit, I expect to see evidence that unmistakably proves it without any doubt. Which you won't have, because it's impossible to prove one way or the other. All we can do is speculate and that's not a valid argument.

**Reply by vvranceanu** [reply]
> > if you're that lost that you obsess about 2 items that give you 2% increased performance each that you think its justified to modify the very structure of the game YOU signed up to play, then go touch grass.
> 
> We aren't talking about me here we are talking about the general player base and this player type exists in the player base. Maybe try to make arguments instead of trying to discredit me as a person.
> 
> > How about, oh, i dont know, not change it then and let people that joined wotlk to get what they expect of a wotlk server? No one joins vanilla servers and start crying there are no emblems to buy gear with.
> 
> This is a nonargument, because different people expect different things and many people specifically asked for the mythic system to come back. No matter what you do some people will be unhappy.

"You" is a form of generalization. I was talking about You, the reader. I don't get how you get offended it by it and not identify with it at the same time. It is what it is i guess.

Also, it is THE argument. Logic cuts both ways. You always favor the least expenditure of resources. It's the intuitive thing to do, since the protozoa. You don't just do something cause it will have net null effect. You first do research, you do cost analysis, you don't just do it cause it might be good, it might be bad... who knows?! If people will always be unhappy, then don't do anything and have the same result. I don't get the problem

**Reply by krutoj87** [reply]
> "You" is a form of generalization. I was talking about You, the reader. I don't get how you get offended it by it and not identify with it at the same time. It is what it is i guess.

Ok but it's still besides the point because the conversation wasn't about what the right or wrong way to play the game is. It was about the statement, that mythic will make more people quit and I pointed out, that there are groups of people for whom the opposite is true. 

> Also, it is THE argument. Logic cuts both ways. You always favor the least expenditure of resources. It's the intuitive thing to do, since the protozoa. You don't just do something cause it will have net null effect. You first do research, you do cost analysis, you don't just do it cause it might be good, it might be bad... who knows?! If people will always be unhappy, then don't do anything and have the same result. I don't get the problem

This starts to go into a completely different direction, than my initial argument. I don't want to debate here if we should or shouldn't implement mythic. I don't need this headache, so I leave this to others. 
All I am saying is: the argument from OP, that mythic will affect the player count negatively can't be proven. It can very well work the other way around.

**Reply by Never6** [reply]
> `On the other hand if people raid the same content for months, for the last elusive items and they never drop, this can also ruin the fun.` if you're that lost that you obsess about 2 items that give you 2% increased performance each that you think its justified to modify the very structure of the game YOU signed up to play, then go touch grass.
> 
> `It's impossible to say if mythic points will affect player retention in a positive or negative way. Both is possible. It's even possible, that it won't affect player retention at all one way or the other. Thus this point is pure speculation and not a real argument.` How about, oh, i dont know, not change it then and let people that joined wotlk to get what they expect of a wotlk server? No one joins vanilla servers and start crying there are no emblems to buy gear with.

1. well some items aren't just a 2% increase, and players wanting max performance of their class is an foreign concept.
2.  So let me get this straight, you join 3.3.5 server  stuck on TBC content,  and you except wrath of the lich king?
If we wanna be true purists, this level 70 twink server shouldn't even have tuning for raids. But that would be very boring

**Reply by ChaoticMe** [reply]
It's a little disingenuous to say "that's not true" because you personally don't believe it. If the number of raid loggers across Retail, Classic, other pservers and this one doesn't indicate to you that it could be true, then maybe try to be more observant and openminded.

**Comment by ChaoticMe** [comment]
I think what can be agreed is that RNG loot sucks. Even if you're not getting the same thing(s) repeatedly, that *one* item you need will not drop, or if it does, the competition is so stiff, you lose. If you spend literal tens/hundreds of hours in a raid, you should feel like it was time well spent and worthwhile. It's hard to feel that way towards gear when it can be hard to get, and also critical to obtain for say tanks, pure DPS, etc. Gearing appears to be really random and that makes us feel mostly powerless, which sucks.

I think it's also agreed that action can be taken to combat the randomness of RNG. Chromie has already done that in a few different ways, including Mythic modes and more. Mythic mode has some limitations, obviously. What else can be done?

There's not a singular answer, imo, but here's what I think will help. A reworking of the Titan Rune currency system from Wrath Classic combined with the current badge vendor path to result in select pieces of gear available for badges. Wrath's framework means everything can drop badges anyway, so we have currency available to use. I don't think this is the best place to fully flesh out the idea, but for example, using the current Mythic system in place, DST and other T4 items would be purchaseable with BoJ. The badge cost per item is heavily negotiable and can be reduced as the tiers progress, if necessary. As with Titan Runes, we could have a smattering of items from raid tiers purchaseable so we're not as subject to RNG. BUT, differently from Wrath Classic, we can ensure the items that need to be purchaseable actually are, for a reasonable amount of currency.

This doesn't solve the "difficulty problem" but I believe that's a separate conversation. Loot and difficulty is connected, most definitely, but muddling the discussion is counterproductive.

-Fuzzbutts
